src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -116,6 +117,56 @@ const SingleContentWrapper = styled.div`
   `}
 `;
 
+const OptionallyRenderedSkipWrapper = ({
```

Comment:
Instead of creating another wrapper for the wrapper in an already oversized container, have we considered using the `SkipLinkWrapper` component directly? For example:

```jsx
const Container = ({ skipLink }) => {

  if (skipLink && skipLink.text) return (
    <SkipLinkWrapper>
      <CpsOnwardJourneyWrapper>
        {/* ... */}
      </CpsOnwardJourneyWrapper>
    </SkipLinkWrapper>
  )
  
  return (
    <CpsOnwardJourneyWrapper>
      {/* ... */}
    </CpsOnwardJourneyWrapper>
  );
}
```

Reply:
:thinking_face: we could do that! but wouldn’t it imply we could literally render the Skip link container anywhere and it wouldn’t visually render as far as there’s no `skipText` prop ?

Reply:
> We could do that! but wouldn’t it imply we could literally render the Skip link container anywhere and it wouldn’t visually render as far as there’s no skipText prop ?

The above example would only render a Skip Link if `skipLink.text` was passed.

I can see you've updated this code, and that it still uses an additional component. The reason I'm adverse to adding a component like this here is because, in my opinion, it makes the container's main render function harder to read. It is a potentially unnecessary abstraction that can be achieved with standard control logic. It also creates another set of defaultProp and propType objects that are unrelated to the component that's exported in this file. Feel free to disagree, because most of these points are stylistic, but they will all play a part in the maintainability of this file in the future.

Reply:
Simon's PR has gone a long way to simplifying this container, so this pattern is less of a concern _at the moment_. I will resolve this. We can revisit later if needed.